src/ui/components/ShapeBase/component.ts

```diff
@@ -0,0 +1,17 @@
+import Component from '@glimmer/component';
+
+export default class ShapeBase extends Component {
+  public shapeId = null;
+
+  constructor(options) {
+    super(options);
+
+    this.shapeId = Math.random()
+      .toString(36)
+      .substr(2, 5);
```

Comment:
Is there a reason why this needs to be limited to 5 characters?
```suggestion
      .substr(2);
```

Reply:
Not really, I just wanted to keep them short.